src/Config/FileLocatorFactoryInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Config;
+
+use Symfony\Component\Config\FileLocatorInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * Interface for file locator factories.
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+interface FileLocatorFactoryInterface
```

Comment:
@aschempp noted that every factory class should have an interface and indeed, every factory in Symfony seems to follow this principle.